gh-issues-import-ng.py

```diff
@@ -13,496 +13,528 @@
+    include_group.add_argument("-i", "--issues", type=int, nargs='+', help="The list of issues to import.")
+
+    args = arg_parser.parse_args()
+
+    def load_config_file(config_file_name):
+        try:
+            config_file = open(config_file_name)
+            config.read_file(config_file)
+            return True
+        except (FileNotFoundError, IOError):
+            return False
+
+    if args.no_config:
+        print("Ignoring default config file. You may be prompted for some missing settings.")
+    elif args.config:
+        config_file_name = args.config
+        if load_config_file(config_file_name):
+            print("Loaded config options from '%s'" % config_file_name)
+        else:
+            sys.exit("ERROR: Unable to find or open config file '%s'" % config_file_name)
+    else:
+        config_file_name = default_config_file
+        if load_config_file(config_file_name):
+            print("Loaded options from default config file in '%s'" % config_file_name)
+        else:
+            print("Default config file not found in '%s'" % config_file_name)
+            print("You may be prompted for some missing settings.")
+
+    if args.username: config.set('login', 'username', args.username)
+    if args.password: config.set('login', 'password', args.password)
+
+    if args.source: config.set('source', 'repository', args.source)
+    if args.target: config.set('target', 'repository', args.target)
+
+    if args.issue_template: config.set('format', 'issue_template', args.issue_template)
+    if args.comment_template: config.set('format', 'comment_template', args.comment_template)
+    if args.pull_request_template: config.set('format', 'pull_request_template', args.pull_request_template)
+
+    # manual setting for TAOS-CI
+    config.set('source', 'username', config.get('source', 'username'))
+    config.set('source', 'password', config.get('source', 'password'))
+    config.set('source', 'server', config.get('source', 'server'))
+    config.set('source', 'repository', config.get('source', 'repository'))
+
+    config.set('target', 'username', config.get('target', 'username'))
+    config.set('target', 'password', config.get('target', 'password'))
+    config.set('target', 'server', config.get('target', 'server'))
+    config.set('target', 'repository', config.get('target', 'repository'))
+
+    config.set('settings', 'import-comments', str(not args.ignore_comments))
+    config.set('settings', 'import-milestone', str(not args.ignore_milestone))
+    config.set('settings', 'import-labels', str(not args.ignore_labels))
+    config.set('settings', 'import-pulls', str(not args.ignore_pulls))
+
+    config.set('settings', 'import-open-issues', str(args.import_all or args.import_open))
+    config.set('settings', 'import-closed-issues', str(args.import_all or args.import_closed))
+
+    # Make sure no required config values are missing
```

Comment:
@pepper-jk I want you to separate this change from this PR. I recommend that you submit new PR with this change (e.g., api_url) . :)

Reply:
What change do you mean? I do not change the API URLs.

Reply:
Any updates?

Reply:
:100: